docs/sources/panels/field-configuration-options.md

```diff
@@ -0,0 +1,228 @@
+title = "Field configuration options"
+type = "docs"
+[menu.docs]
+parent = "panels"
+weight = 300
++++
+
+# Field configuration options
+
+This page explains what field configurations and field overrides in Grafana are and how to use them. It also includes [examples](#examples) if you need an idea of how this feature might be useful in the real world.
+
+> **Note:** This documentation refers to a Grafana 7.0 beta feature. This documentation will be frequently updated to reflect updates to the feature, and it will probably be broken into smaller sections when the feature moves out of beta.
+
+The data model behind Grafana, the [data frame]({{< relref "../developers/plugins/data-frames.md" >}}), is a columnar-oriented table structure. 
+
+Each column within this structure is called a _field_. Grafana allows to customize how a particular field is displayed in the visualization.
+
+## Field configuration options and overrides
+
+Field configuration options allow you to change how the data is displayed in your visualizations. Options and overrides that you apply do not change the data, they change how Grafana displays the data.
+
+> **Note:** The time fields are not affected by field configuration options or overrides.
+
+_Field configuration options_, both standard and custom, are applied in the **Field** tab in the panel editor. Changes on this panel apply to all fields in the visualization. For example, if you change the unit to percentage, then all fields with numeric values will be calculated in percentages. [Apply a field option](#configure-all-fields).
+
+_Field overrides_ are applied in the _Overrides_ tab in the panel editor. They are exactly the same as field configuration options except that they only change fields you select. The current feature only allows you to change one field at a time, but future improvements will offer more flexibility. [Apply an override](#override-a-field).
+
+All [field options](#field-options) are defined below.
+
+### Standard field options
+
+Standard field options are:
+
+- [Unit](#unit)
+- [Min](#min)
+- [Max](#max)
+- [Decimals](#decimals)
+- [Display name](#display-name)
+- [No value](#no-value)
+- [Thresholds](#thresholds)
+- [Value mappings](#value-mapping)
+- [Data links](#data-links)
+
+You can apply standard field options to the following panel visualizations:
+
+- Bar gauge
+- Gauge
+- Stat
+- Table
+
+### Custom field options
+
+You can only apply custom field options to table visualizations.
+
+Custom field options are:
+
+- [Column width](#column-width)
+- [Column alignment](#column-alignment)
+- [Cell display mode](#cell-display-mode)
+
+## Configure all fields
+
+To change how all fields display data, you apply a [field option](#field-options). Usually you apply changes that you want to most or all of the fields here, than apply field overrides to exceptions.
+
+1. Navigate to the panel you want to edit, click the panel title, and then click **Edit**.
+2. Click the **Field** tab.
+3. Find the 
```

Comment:
?

Reply:
Diana was tired and lost her train of thought.

Reply:
yes, that's tru